packages/admin/cms-admin/src/pages/pageTree/useCopyPastePages.tsx

```diff
@@ -292,17 +344,37 @@ function useCopyPastePages(): UseCopyPastePagesApi {
             // 2. Refetch Pages query
             client.refetchQueries({ include: ["Pages"] });
         },
-        [client, documentTypes, scope],
+        [client, damScope, documentTypes, scope],
     );
 
     return { prepareForClipboard, writeToClipboard, getFromClipboard, sendPages };
 }
 
+function mergeIdsMaps(map1: IdsMap, map2: IdsMap): IdsMap {
```

Comment:
As discussed in JFX: Please use two separate IDs maps `damFileIdsToReplace` and `pageTreeNodeIdsToReplace`